integration_tests/mockApis/ciagInductionUi.ts

```diff
@@ -0,0 +1,18 @@
+import { stubFor } from './wiremock'
+
+const stubCreateCiagInductionUi = (prisonNumber = 'G6115VJ') =>
```

Comment:
I've had to create a stub to represent the CIAG Induction UI for when we click "make a progress plan" on our (pre-induction) overview page. This is just a simple html page that represents the CIAG Induction UI